pyincore/analyses/epfrestoration/epfrestoration.py

```diff
@@ -9,6 +9,7 @@
 
```

Comment:
Not related to this PR, but can you add the copyright 2022 line to the header for epfrestoration?